e2e/scripts/pageHelpers.js

```diff
@@ -254,6 +282,43 @@ export const loginShopper = async ({page, userCredentials}) => {
     }
 }
 
+/**
+ * Attempts to log in a shopper with provided user credentials.
+ * 
+ * @param {Object} options.page - Object that represents a tab/window in the browser provided by playwright
+ * @return {Boolean} - denotes whether or not login was successful
+ */
+export const socialLoginShopper = async ({page}) => {
+    try {
+        await page.goto(config.RETAIL_APP_HOME + "/login");
+
+        await page.getByRole("button", { name: /Google/i }).click();
+        await expect(page.getByText(/Sign in with Google/i)).toBeVisible({ timeout: 10000 });
+        await page.waitForSelector('input[type="email"]');
+
+        // Fill in the email input
+        await page.fill('input[type="email"]', 'e2e.pwa.kit@gmail.com');
+        await page.click('#identifierNext');
+
+        await page.waitForLoadState();
+
+        // Fill in the password input
+        await page.fill('input[type="password"]', 'hpv_pek-JZK_xkz0wzf');
```

Comment:
I wonder if this is going to get caught by security? @shethj do we have an precedence for this, are we using secrets and environment variables for anything that we can copy the implementation?

@yunakim714 can you reach out to jainam for this.. I think he's knowledgable in the e2e landscape.

Reply:
Yeah we set all secrets in Github Actions settings and they can be set as env vars at runtime. 
Values can be read from process.env in the tests.
Refer this code where we read the storefront domain from env var: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/57c24e1eab035519aa62816b0a9d798e6a5cbb1a/e2e/config.js#L10

Reply:
Added the credentials as GitHub env vars !